### PR TITLE
Add Probe Control Enabler

### PR DESCRIPTION
**Pull request by Felger**

### CODE REVIEW

File: NetKAN/RemoteTech-ProbeControlEnabler.netkan [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Felger/FelgerFixes",
+	"ksp_version"    : "1.0"
```
Comment (dbent) [inline NetKAN/RemoteTech-ProbeControlEnabler.netkan]: Needs a comma here.

File: NetKAN/RemoteTech-ProbeControlEnabler.netkan [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "identifier"     : "RemoteTech-ProbeControlEnabler",
+    "abstract"       : "Optional ModuleManager patch by Felger to re-enable probe control when a link is not present.  You will still need a link to transmit science data.",
+    "license"        : "CC-BY-SA",
+    "release_status" : "stable",
+    "depends" : [
+        { "name" : "ModuleManager" }
+    ],
+    "install" : [
+        {
+            "find"       : "RT_ProbeRemover.cfg",
```
Comment (dbent) [inline NetKAN/RemoteTech-ProbeControlEnabler.netkan]: Same as the other PR:

`find` doesn't work on files, only directories (although I see no reason why it shouldn't also work on files). You'll need:
```
"file"       : "GameData/ZZZ_FelgerFixes/RT_ProbeRemover.cfg",
```

**Comment by Felger** [comment]
I believe that'll also reduce the spec_version, but I don't recall which spec version I ought to drop it to at this point, besides, folks should have that updated to the latest version!

**Comment by dbent** [comment]
`v1.2` but that's fine, I'll update it if a 1.2 user raises an issue.

Tested manually, merging.

**Pull request merged**